src/UI/admin/components/hall-request/AdminHallRequestStyle.js

```diff
@@ -11,4 +11,504 @@ const HallReqTitle = styled.h2`
   color: blue;
 `;
 
-export { HallReqContainer, HallReqTitle };
+const HallDiv = styled.div`
+  margin: auto;
+  width: 1024px;
+  height: 352px;
```

Comment:
ovo ti brejka componentu prva u zeplinu je 352px druga je 392 treca je 296 nesmijes definirati height to sama komponenta odlucuje po contentu